stable

```diff
@@ -0,0 +1 @@
+main
```

Comment:
we'll add the `stable` symlink when the time comes but for now it shouldn't point to main - do you mind removing this?

Reply:
I can remove but the index.html won't work then.